store/storetest/team_store.go

```diff
@@ -536,20 +536,17 @@ func testGetAllTeamPageListing(t *testing.T, ss store.Store) {
 	_, err = ss.Team().Save(&o4)
 	require.Nil(t, err)
 
-	if r1 := <-ss.Team().GetAllTeamPageListing(0, 10); r1.Err != nil {
-		t.Fatal(r1.Err)
-	} else {
-		teams := r1.Data.([]*model.Team)
+	teams, err := ss.Team().GetAllTeamPageListing(0, 10)
+	require.Nil(t, err)
```

Comment:
Nit: `require.NoError` is more appropriate here. 0/5

Reply:
No is not, it generates certain problems related to we don't return an error, we return a model.AppError, which is not exactly the same.

Reply:
Ouch. Thanks for clarifying!